### DISCUSSION TITLE
Amoro 能够完全替代hms？

### DISCUSSION THREAD

**Discussion by melin** [apache/amoro#2686] [Q&A]
Amoro 兼容hms？ 支持多多种类型表格式的元数据存储？ parquet, orc, hudi，paimon 等表

**Comment by zhoujinsong** [comment]
I don't think AMS can completely replace HMS, and that wasn't the goal when AMS was designed.
我不认为AMS可以完全替代HMS，AMS在设计之初也并没有这个目标。

HMS can be registered as an external catalog into AMS, and further, AMS can automatically manage/optimize the Iceberg/Paimon tables within this HMS. But this only applies to Iceberg/Paimon tables. AMS doesn't manage other tables within HMS.
HMS可以作为一个外部catalog，注册到AMS中，进一步AMS可以自动管理/优化这个HMS中的iceberg/paimon表。但是也仅针对Iceberg/Paimon表而已，AMS并不管理HMS中的其他表。

At the same time, in some scenarios, you can use the internal catalog provided by AMS directly as the metadata store for Iceberg tables. This means you no longer need to use HMS as the metadata store for Iceberg tables.
同时在一些场景可以使用AMS提供的internal catalog直接作为iceberg表的元数据中心，这样你就不再需要使用HMS作为iceberg表的元数据中心了。

**Reply by melin** [reply]
有时候数据湖表和hive 普通混合使用，可能ods 层有一些iceberg/hudi 表，存在多个cataog，spark 访问表，是否会对数仓人员增加使用难度？

**Reply by baiyangtx** [reply]
可以看看目前 amoro 项目中的 unified catalog 。它的目的是提供一个可以访问不同 format 的 catalog 实现。
